client/jetpack-cloud/sections/partner-portal/primary/issue-license/index.tsx

```diff
@@ -87,11 +87,7 @@ export default function IssueLicense( { selectedSite, suggestedProduct }: Assign
 
 	// If URL params are present, use them to open the review licenses modal directly.
 	useEffect( () => {
-		if (
-			getQueryArg( window.location.href, 'source' ) === 'manage-pricing-page' &&
-			getQueryArg( window.location.href, 'product_slug' ) !== 'jetpack-backup-t2' &&
-			getQueryArg( window.location.href, 'product_slug' ) !== 'jetpack-security-t2'
-		) {
+		if ( getQueryArg( window.location.href, 'source' ) === 'manage-pricing-page' ) {
```

Comment:
In this file, just below (on line 96, so could not select it here in Github), we also use 'product_slug', which ought to be changed to 'products'.